--- qbittorrent-3.2.0/src/app/qtsingleapplication/qtlocalpeer.cpp.omv~	2015-06-08 17:36:47.985639173 +0200
+++ qbittorrent-3.2.0/src/app/qtsingleapplication/qtlocalpeer.cpp	2015-06-08 17:36:58.780521768 +0200
@@ -41,6 +41,7 @@
 
 #include "qtlocalpeer.h"
 #include <QCoreApplication>
+#include <QDataStream>
 #include <QTime>
 
 #if defined(Q_OS_WIN)
--- qbittorrent-3.2.0/src/core/qtlibtorrent/alertdispatcher.cpp.omv~	2015-06-08 17:39:00.144238399 +0200
+++ qbittorrent-3.2.0/src/core/qtlibtorrent/alertdispatcher.cpp	2015-06-08 17:39:13.934096445 +0200
@@ -30,6 +30,8 @@
 
 #include "alertdispatcher.h"
 
+#define BOOST_NO_CXX11_RVALUE_REFERENCES 1
+
 #include <libtorrent/session.hpp>
 #include <boost/bind.hpp>
 #include <QMutexLocker>
